tests/js/utils/datetime.js

```diff
@@ -0,0 +1,35 @@
+
+import Datetime from 'utils/datetime';
+
+
+test('Datetime constructor', () => {
+    let dt = new Datetime(0);
+
+    // this.datetime is set with a new Date
+    expect(dt.datetime).toEqual(new Date(0));
+
+    // default locale is en-GB
+    expect(dt.locale).toBe('en-GB');
+
+    // date and time formatting is specified
+    expect(dt.dateFormat).toEqual({
+        day: 'numeric',
+        month: 'long',
+        year: 'numeric'})
+    expect(dt.timeFormat).toEqual({
+        hour: '2-digit',
+        minute: '2-digit',
+        second: '2-digit'})
```

Comment:
These two assertions are basically testing that you didn't make a spelling mistake in your code. I really fail to see what value this provides over just proof-reading your code (instead of writing it down twice).

Reply:
im in the DRY-except-in-unit-tests school of thought here (see here for interesting discussion http://codebetter.com/karlseguin/2009/09/12/unit-testing-do-repeat-yourself/)

i would view this kind of test as more harness than logic check

Reply:
This is not about repeating code, it's about the test doing nothing but verifying that you, the developer, didn't make a spelling mistake. It's not testing a feature, or a side-effect, it's just checking you wrote the right thing by writing that same thing again. 

If I try to simplify it, it's the same as doing this:
```javascript
var answer = 42;

test('answer', () => {
    expect(answer).toEqual(42);
});
```

And I honestly do not see what value that adds.

Reply:
>  it's about the test doing nothing but verifying that you, the developer, didn't make a spelling mistak

its about future you, me and others not messing with this unless they are prepared to re/move/update tests

Reply:
This however is a blocker for me.

Reply:
how do you propose to test this - or are you suggesting we dont ?

Reply:
Yep, I think we should just not have those 2 tests.

Reply:
i really dont agree - and im tempted to refactor the class to get implict coverage, but pretty reluctant to do that.

Ill remove these tests for the sake of getting this landed